pkg/adapterManager/manager_test.go

```diff
@@ -38,12 +43,51 @@ type (
 	fakeevaluator struct {
 		expr.Evaluator
 	}
+
+	testManager struct {
+		name     string
+		throw    bool
+		instance testAspect
+	}
+
+	testAspect struct {
+		throw bool
+	}
 )
 
 func (m *fakemgr) Kind() string {
 	return m.kind
 }
 
+func newTestManager(name string, throwOnNewAspect bool, aspectThrowOnExecute bool) testManager {
```

Comment:
Do we have a manager/aspect for testing already? If not, we should create one (I'll need to another to use in a different package as part of testing the pkg/api surface in a PR coming soon).